relays/substrate/src/main.rs

```diff
@@ -934,4 +934,40 @@ mod tests {
 		);
 		assert!(Rialto::verify_message(&payload).is_err());
 	}
+
+	#[test]
+	fn rialto_tx_extra_bytes_constant_is_correct() {
```

Comment:
```suggestion
	fn rialto_tx_extra_bytes_constant_is_correct() {
```

:heart: Thanks for this, in the instant I saw `103` hardcoded I started writing a comment that we need a test to know if it breaks :)